proto/poktroll/supplier/tx.proto

```diff
@@ -9,33 +10,32 @@ import "cosmos/msg/v1/msg.proto";
 import "cosmos_proto/cosmos.proto";
 import "gogoproto/gogo.proto";
 import "cosmos/base/v1beta1/coin.proto";
-
 import "poktroll/supplier/params.proto";
 import "poktroll/shared/service.proto";
 
-
 // Msg defines the Msg service.
 service Msg {
   option (cosmos.msg.v1.service) = true;
-
+  
   // UpdateParams defines a (governance) operation for updating the module
   // parameters. The authority defaults to the x/gov module account.
   rpc UpdateParams    (MsgUpdateParams   ) returns (MsgUpdateParamsResponse   );
   rpc StakeSupplier   (MsgStakeSupplier  ) returns (MsgStakeSupplierResponse  );
   rpc UnstakeSupplier (MsgUnstakeSupplier) returns (MsgUnstakeSupplierResponse);
+  rpc UpdateParam     (MsgUpdateParam    ) returns (MsgUpdateParamResponse    );
 }
 // MsgUpdateParams is the Msg/UpdateParams request type.
 message MsgUpdateParams {
-  option (cosmos.msg.v1.signer) = "authority";
-  option (amino.name) = "poktroll/x/supplier/MsgUpdateParams";
-
+  option (cosmos.msg.v1.signer) =                           "authority";
+  option           (amino.name) = "poktroll/x/supplier/MsgUpdateParams";
+  
   // authority is the address that controls the module (defaults to x/gov unless overwritten).
   string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
 
   // TODO_IMPROVE(#322): The requirement to provide all params is adopted from the
   // latest Cosmos SDK version. We should look into either improving this ourselves
   // or seeing if it is on their roadmap.
-
+  
```

Comment:
```suggestion

```

Reply:
I think this comment is actually resolved at this point. Removing.